cgmanifest.json

```diff
@@ -10571,8 +10571,8 @@
         "type": "other",
         "other": {
           "name": "libxml2",
-          "version": "2.9.14",
-          "downloadUrl": "https://gitlab.gnome.org/GNOME/libxml2/-/archive/v2.9.14/libxml2-v2.9.14.tar.gz"
+          "version": "2.10.0",
+          "downloadUrl": "https://gitlab.gnome.org/GNOME/libxml2/-/archive/v2.9.14/libxml2-v2.10.0.tar.gz"
```

Comment:
this URL looks incorrect

Reply:
I think it should be: https://gitlab.gnome.org/GNOME/libxml2/-/archive/v2.10.0/libxml2-v2.10.0.tar.gz

Reply:
good catch